e2e/custom-test-sequencer/testSequencer.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const Sequencer = require('@jest/test-sequencer').default;
+const path = require('path');
+
+const priorityOrder = [
+  path.resolve('./d.test.js'),
+  path.resolve('./b.test.js'),
+  path.resolve('./c.test.js'),
+];
+const FAIL = 0;
+
+class CustomSequencer extends Sequencer {
+  sort(tests) {
```

Comment:
way better now, thanks! I still think this does more than it have to, though. Why not just sort alphabetically?

Reply:
Nice!
Sort alphabetically make it more simple and easy to understand.
Thanks.